07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -12,7 +12,7 @@ h1 {
   color: white;
```

Comment:
Thank you, completed

 

From: Ethan Stenis [mailto:notifications@github.com] 
Sent: Tuesday, August 23, 2016 3:11 PM
To: AustinCodingAcademy/intro-workbook <intro-workbook@noreply.github.com>
Cc: Steve Anderson <anderson-steven@att.net>; Author <author@noreply.github.com>
Subject: Re: [AustinCodingAcademy/intro-workbook] Austin.Intro.2.Anderson.Steve (#84)

 

In 07LessonSeven/in-class-project/css/my-style.css <https://github.com/AustinCodingAcademy/intro-workbook/pull/84#discussion_r75941311> :

> @@ -12,7 +12,7 @@ h1 {
>    color: white;

Try and center your NASA page with some style on the body - margin: 0 auto; and width: 90%;.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/AustinCodingAcademy/intro-workbook/pull/84/files/db68edff3ae26773c60869231264d125308c18a6..a9a6210ed3d024ee1f675739f39535d00a7e2c5f#r75941311> , or mute the thread <https://github.com/notifications/unsubscribe-auth/ATJFMG5Jb7vETSZLACnj2cDbEmOZLg80ks5qi1PmgaJpZM4JVvJX> .  <https://github.com/notifications/beacon/ATJFMDjjpfbRE-GBC6Z-zVMFdnATzF4-ks5qi1PmgaJpZM4JVvJX.gif>